### PR TITLE
Android layout

### PR DESCRIPTION
**Pull request by Anna-Timoshenko**

### CODE REVIEW

File: HelloDroid/Views/NavigationActivity.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+using Android.Widget;
+using Hello.ViewModels;
+using Softeq.XToolkit.Bindings;
+using Softeq.XToolkit.WhiteLabel;
+using Softeq.XToolkit.WhiteLabel.Droid;
+
+namespace HelloDroid.Views
+{
+    [Activity]
+    class NavigationActivity : ActivityBase<NavigationViewModel>
```
Comment (TimofeyBurak) [inline HelloDroid/Views/NavigationActivity.cs]: Access modifier is missing

File: Hello/ViewModels/NavigationViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    {
+                        _pageNavigationService
+                            .For<AddCollarPairViewModel>()
+                            .Navigate();
+                    }),
+                    "Go Add")),
+            };
+
+
+            //Items.AddRangeToGroups<(Category Header, CommandAction Item)>(actions , x => x.Header.ToString(), x => x.Item);
```
Comment (TimofeyBurak) [inline Hello/ViewModels/NavigationViewModel.cs]: Commented code should not be commited

**Review by TimofeyBurak — CHANGES_REQUESTED** [review]

File: HelloDroid/Views/NavigationActivity.cs (hunk 1 above)
Comment (Anna-Timoshenko) [reply HelloDroid/Views/NavigationActivity.cs]: fixed

File: Hello/ViewModels/NavigationViewModel.cs (hunk 2 above)
Comment (Anna-Timoshenko) [reply Hello/ViewModels/NavigationViewModel.cs]: fixed

**Review by TimofeyBurak — APPROVED** [review]

**Pull request merged**